dcp_diag/component_agents/azul_agent.py

```diff
@@ -13,14 +13,24 @@ def __init__(self, deployment):
     def get_project_bundle_fqids(self, document_id, page_size=1000):
         bundle_fqids = set()
 
-        filter_dict = urllib.parse.quote('{"file":{"projectId":{"is":["' + document_id + '"]}}}')
-        base_url = self.azul_service_url + f'/repository/files?filters={filter_dict}&order=desc&sort=entryId'
-        url = base_url + f'&size={page_size}&order=desc'
+        filters = {
+            'projectId': {
+                'is': [
+                    document_id
+                ]
+            }
+        }
+        params = {
+            'filters': json.dumps(filters),
+            'size': page_size
+        }
+        base_url = self.azul_service_url + f'/repository/bundles'
+        url = base_url + f'&size={page_size}'
```

Comment:
Is the `+ f'&size={page_size}'` necessary?
Seems like size gets into params on line 25.

Reply:
Actually no, it's not needed. Thanks for catching that.